pkg/hpsf/hpsfTypes.go

```diff
@@ -17,16 +19,46 @@ type ConnectionType string
 
 const (
 	CTYPE_UNKNOWN ConnectionType = "unknown"
-	CTYPE_TRACES  ConnectionType = "OTelTraces"
 	CTYPE_LOGS    ConnectionType = "OTelLogs"
 	CTYPE_METRICS ConnectionType = "OTelMetrics"
-	CTYPE_EVENT   ConnectionType = "OTelEvent"
-	CTYPE_HONEY   ConnectionType = "Honeycomb"
+	CTYPE_TRACES  ConnectionType = "OTelTraces"
+	CTYPE_EVENTS  ConnectionType = "OTelEvents"
+	CTYPE_HONEY   ConnectionType = "HoneycombEvents"
 	CTYPE_NUMBER  ConnectionType = "number"
 	CTYPE_STRING  ConnectionType = "string"
 	CTYPE_BOOL    ConnectionType = "bool"
 )
 
+// These are the possible connection types that can be used in the HPSF pipelines.
+var PipelineSignalTypes = []ConnectionType{
```

Comment:
We've deviated from connection to signal type in these new variables and I found it to be confusing. Can we try to consolidate on a single term? I'm okay with either Connection or Signal.

Reply:
Good point. Connection is baked into HPSF so I'll keep that one.